.travis.yml

```diff
@@ -2,8 +2,6 @@ language: node_js
 node_js:
   - node
 
-sudo: required
```

Comment:
Why remove `sudo: required`? I see that it should provide faster boot times but less memory (which I guess isn't an issue).
https://docs.travis-ci.com/user/ci-environment/#Virtualization-environments